src/notifications/ns_settings.py

```diff
@@ -43,21 +43,26 @@ def __init__(self, client):
     def loadSettings(self):
         self.enabled = Settings.get('notifications/enabled', True, type=bool)
         self.popup_lifetime = Settings.get('notifications/popup_lifetime', 5, type=int)
+        self.popup_position = Settings.get('notifications/popup_position', "bottom right", type=str)
 
         self.nsEnabled.setChecked(self.enabled)
         self.nsPopLifetime.setValue(self.popup_lifetime)
+        comboboxIndex = [i for i in range(4) if self.nsPositionComboBox.itemText(i) == self.popup_position]
+        self.nsPositionComboBox.setCurrentIndex(comboboxIndex[0] if comboboxIndex else -1)
```

Comment:
But 0 is falsey, so if the combobox index is 0 this won't work as expected (and in any case this list comprehension is a slightly strange way of doing this).